movai_core_shared/core/zmq_client.py

```diff
@@ -165,7 +165,7 @@ def _init_context(self):
         self._zmq_ctx = zmq.asyncio.Context()
 
     def _init_lock(self):
-        self._lock = asyncio.Lock()
+        pass
```

Comment:
remove this function 
and add `self._lock = Lock` to init_context 
that is different in each class